windows/window_manager_plugin.cpp

```diff
@@ -116,8 +116,6 @@ std::optional<LRESULT> WindowManagerPlugin::HandleWindowProc(HWND hWnd,
     // This must always be first or else the one of other two ifs will execute
     //  when window is in full screen and we don't want that
     if (wParam && window_manager->IsFullScreen()) {
-      NCCALCSIZE_PARAMS* sz = reinterpret_cast<NCCALCSIZE_PARAMS*>(lParam);
-      sz->rgrc[0].bottom -= 3;
       return 0;
     }
 

```

Comment:
While we're at it, can we add one more condition to ignore `WM_NCCALCSIZE` handling?

It won't break the plugin; we are implementing our in-house fullscreen mechanism in [`package:media_kit`](https://github.com/alexmercerind/media_kit/pull/250); and it will be great to have it compatible with [`package:window_manager`](https://github.com/leanflutter/window_manager).

```suggestion

    if (!(GetWindowLongPtr(hWnd, GWL_STYLE) & WS_OVERLAPPEDWINDOW))
      return 0;
    }
```